gloo/docs/user_guides/gateway/configuring_route_options/advanced_routing_action.md

```diff
@@ -195,11 +203,134 @@ apiVersion: gateway.solo.io/v1
               namespace: gloo-system
 {{< /highlight >}}
 
-### Multiple Destination {#multi}
+
+#### Kubernetes destination {#dest-kube}
+
+If you are running Gloo in a Kubernetes cluster, it is possible to directly specify 
+[Kubernetes Services](https://kubernetes.io/docs/concepts/services-networking/service/) as routing destinations. 
+The `kube` destination type has two required fields: `ref` is a [ResourceRef]({{% ref "/v1/github.com/solo-io/solo-kit/api/v1/ref.proto.sk#resourceref" %}}) 
+to the service that should receive traffic and `port` is the port on which the service is listening.
+
+The following configuration will forward all requests to `/petstore` to port `8080` on the Kubernetes service named 
+`petstore` in the `default` namespace.
+
+{{< highlight yaml "hl_lines=6-10" >}}
+routes:
+- matcher:
+    prefix: /petstore
+  routeAction:
+    single:
+      kube:
+        ref:
+          name: petstore
+          namespace: default
+        port: 8080
+{{< /highlight >}}
+
+#### Consul destination {#dest-consul}
+
+Gloo is capable of discovering services registered with [HashiCorp Consul](https://www.hashicorp.com/products/consul/). 
+If this feature has been enabled via the [correspondent field]({{% ref "/v1/github.com/solo-io/gloo/projects/gloo/api/v1/settings.proto.sk#servicediscoveryoptions" %}}) 
```

Comment:
I'd suggest linking to the ConsulConfiguration (http://localhost:1313/v1/github.com/solo-io/gloo/projects/gloo/api/v1/settings.proto.sk/#consulconfiguration) and calling out or separately linking to the service discovery option. I clicked through the link and I landed in the middle of a table of settings and I had no idea what I was supposed to be looking at. I would not assume the reader understands about Gloo upstream discovery.

Reply:
Done